app/views/agencies/show.html.erb

```diff
@@ -5,11 +5,14 @@
 
 <span  class="page-header articleTitle">
   <h1 class="page-header articleTitle"><%= @agency.name %></h1>
-  <h4><%= @agency.street_address %></h4>
-  <h4><%= @agency.city %>, <%= State.find(@agency.state_id).name %> <%= @agency.zipcode %></h4>
```

Comment:
Sounds simple, but I need more clarification. Would you like for me to have two h4's as in the old code, and replace the State name evaluation with a scope?